src/components/override-sdk/infra/Assignment/Assignment.tsx

```diff
@@ -292,12 +293,29 @@ export default function Assignment(props) {
     });
   }
 
+  function handleBackLinkforDateComp(){
+    const childPconnect = children[0]?.props?.getPConnect();
+    const dateField = PCore.getFormUtils().getEditableFields(childPconnect.getContextName()).filter(field => field.type.toLowerCase() === 'date');
+    if(dateField){
+      dateField?.forEach(field => {
+        const childPagRef = children[0]?.props?.getPConnect().getPageReference();
```

Comment:
_children[0]?.props?.getPConnect()_ can be replaced with _childPconnect_

Reply:
Its replaced now.